.husky/pre-commit

```diff
@@ -1,6 +1,3 @@
-#!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
-
```

Comment:
These lines were generating a deprecation warning. I verified that with them removed the warning went away and the pre-commits still executed.